cluster-autoscaler/cloudprovider/gce/cache.go

```diff
@@ -397,45 +392,39 @@ func (gc *GceCache) InvalidateAllMigInstanceTemplates() {
 }
 
 // GetMachineFromCache retrieves machine type from cache under lock.
-func (gc *GceCache) GetMachineFromCache(machineType string, zone string) (*gce.MachineType, error) {
+func (gc *GceCache) GetMachineFromCache(machineTypeName string, zone string) (MachineType, bool) {
```

Comment:
nit: `FromCache` and `ToCache` suffixes are really redundant here.

Reply:
done